google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ReadWriteTransaction.java

```diff
@@ -254,19 +265,30 @@ public Timestamp getReadTimestampOrNull() {
     return null;
   }
 
-  private boolean hasCommitTimestamp() {
-    return commitTimestampFuture != null;
+  private boolean hasCommitResponse() {
```

Comment:
I originally thought this meant the response wasn't available yet, but if it means this object doesn't support commit responses, then I'm again suspecting the API needs some further thought.

Reply:
For this class your initial thought is correct. This indicates that the commit response is not yet available, because the transaction has not yet committed. Once the transaction has been committed, a commit response will be available.